src/scripts/content/atlassian.js

```diff
@@ -28,14 +28,18 @@ togglbutton.render('#ghx-detail-view [spacing] h1:not(.toggl)', {observe: true},
   numElem.parentNode.appendChild(container);
 });
 
-// Jira 2017 sprint modal
-togglbutton.render('#ghx-issue-fragment [spacing] a img:not(.toggl)', {observe: true}, function () {
+
+// Jira 2018 sprint modal
+// Using the h1 as selector to make sure that it will only try to render the button
+// after Jira has fully rendered the modal content
+togglbutton.render('div[role="dialog"] h1:not(.toggl)', {observe: true}, function (titleElem) {
+  // need to all the way up until we find the modal wrapper
+  var rootElem = titleElem.offsetParent.offsetParent.offsetParent.offsetParent
```

Comment:
Do we really need to use `offsetParent` instead of `parentNode`? `offsetParent` seems too dependant on elements being positioned.

Reply:
The idea is to get to the dialog root element as fast as we can. I could change to use CSS selectors instead but looks like Jira is always changing the elements classes. :cry: